addons/html_editor/static/src/others/embedded_components/plugins/file_plugin/file_blueprint.xml

```diff
@@ -1,6 +0,0 @@
-<templates>
-    <t t-name="html_editor.EmbeddedFileBlueprint">
-        <div t-att-data-embedded-props="embeddedProps" data-embedded="file"
-            data-oe-protected="true" contenteditable="false"/>
-    </t>
-</templates>
```

Comment:
Stable retrocompatibility: put back this file with a comment explaining it's not used anymore but is kept due to the stable policy.

Reply:
It's a template that was moved to another directory, so the template is still there and does not break compatibility.